src/endpoints/location.js

```diff
@@ -6,10 +6,12 @@ export default new Endpoint(
   'locations',
   'https://cms.integreat-app.de/wp-json/extensions/v1/multisites',
   json => {
+    if (!json) {
+      return {}
```

Comment:
whoops yeah that's right, don't like it though - this sorts the array where it should only load an array in. Sorting should happen somewhere else, but yeah... :(